pulp_ansible/app/viewsets.py

```diff
@@ -195,6 +220,27 @@ class CollectionVersionViewSet(UploadGalaxyCollectionMixin, SingleArtifactConten
     filterset_class = CollectionVersionFilter
     ordering_fields = ("pulp_created", "name", "version", "namespace")
 
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": "create",
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_required_repo_perms_on_upload:ansible.modify_ansible_repo_content",
```

Comment:
repo level permissions don't make sense here because these apis aren't scoped to a repository to make changes on. Its probably okay to just use the v3 apis for creating and modifying collections and reserve these apis for admin users (same goes for the other content views in this file)

Reply:
This is a Pulp RBAC design choice. When a plugin adds RBAC, content becomes accessible only through permissions on repositories. This includes uploads, you can only upload to repositories you have permission for. The reason being that users can only see content for repositories they have permissions on, so they can not see orphan content unless they are an admin. So now if you are uploading to this endpoint and are a non-admin you must specify the repository to upload to or you will get a 403.